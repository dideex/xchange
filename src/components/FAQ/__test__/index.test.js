import React from 'react'
import Component from '../index'
import {shallow} from 'enzyme'

describe('FAQ tests', () => {
  describe('Markup', () => {
    it('Basic markup', () => {
      const wrapper = shallow(<Component />)
      expect(wrapper.html()).toMatchSnapshot()
    })
    it('After collapsing block', () => {
      const wrapper = shallow(<Component />)
      wrapper.find('.currency-item').first().simulate('click')
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})