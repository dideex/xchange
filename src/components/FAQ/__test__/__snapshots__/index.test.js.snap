// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQ tests Markup After collapsing block 1`] = `"<div class=\\"css-r00eij\\"><h2 class=\\"css-x184e\\">FAQ</h2><div class=\\"css-1x3ooie\\"><div rotate=\\"0deg\\" class=\\"css-pvdp3k\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #1</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div><div class=\\"css-1lkfryp\\">The nature of promises that they remain to immune changing circamstences</div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #2</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #3</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #4</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #5</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #6</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #7</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #8</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div></div></div>"`;

exports[`FAQ tests Markup Basic markup 1`] = `"<div class=\\"css-r00eij\\"><h2 class=\\"css-x184e\\">FAQ</h2><div class=\\"css-1x3ooie\\"><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #1</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #2</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #3</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #4</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #5</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #6</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #7</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div><div rotate=\\"-90deg\\" class=\\"css-fr62av\\"><div class=\\"currency-item\\"><div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">Question #8</div></div><svg style=\\"width:32px;height:32px\\" class=\\"\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill:#aee239\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div></div></div>"`;
